### PR TITLE
feature/SDK-30_default-pd-format

### PR DESCRIPTION
**Pull request by sanderPostma**

### CODE REVIEW

File: packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ export class OID4VP {
       context: this.session.context,
       domain: proofOptions.domain,
       challenge: proofOptions.challenge,
-      format: opts?.restrictToFormats ?? selectedVerifiableCredentials.definition.definition.format,
+      fallbackFormat: opts?.restrictToFormats ?? selectedVerifiableCredentials.definition.definition.format,
```
Comment (nklomp) [inline packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts:159]: This is no fallback format

File: packages/siopv2-oid4vp-op-auth/src/session/functions.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ export async function createOID4VPPresentationSignCallback({
       fetchRemoteContexts,
       domain,
       challenge,
-      format,
+      fallbackFormat: fallbackFormat,
```
Comment (nklomp) [inline packages/siopv2-oid4vp-op-auth/src/session/functions.ts:50]: Format not fallback

File: packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts (hunk 1 above)
Comment (sanderPostma) [reply packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts:159]: I'll rename it back, but it's the last option in 
```
 const formatOptions = format ?? presentationDefinition.format ?? args.fallbackFormat
```

Comment (nklomp) [reply packages/siopv2-oid4vp-op-auth/src/session/OID4VP.ts]: Yes which results in a format being used afterwards that is not a fallback in the end of course

File: packages/siopv2-oid4vp-op-auth/src/session/functions.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export async function createOID4VPPresentationSignCallback({
   challenge?: string
   fetchRemoteContexts?: boolean
   skipDidResolution?: boolean
-  format?: Format
+  fallbackFormat?: Format
```
[the remark this answers is not in the document]
Comment (sanderPostma) [reply packages/siopv2-oid4vp-op-auth/src/session/functions.ts]: fixed

File: packages/siopv2-oid4vp-op-auth/src/session/functions.ts (hunk 2 above)
Comment (sanderPostma) [reply packages/siopv2-oid4vp-op-auth/src/session/functions.ts]: reverted

**Comment by sanderPostma** [comment]
@nklomp 
Shouldn't we select a format from opSession.authenticationRequest.vp_formats ? When it supports vc+sd-jwt we can take select that, otherwise another one.

**Comment by nklomp** [comment]
Yes that is correct. But we should already be doing that I think.

**Pull request closed without merging**